src/react-components/src/components/01-atoms/Forms/Checkbox.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * A Form Checkbox component that can be selected and deselected.
+ */
```

Comment:
It might be better to have these comments right before the component itself rather than at the top of the file. Not a big deal but we should have a convention, especially when this very closely relates to the component.